libraries/Microsoft.Bot.Builder.Dialogs.Adaptive.Testing/Mocks/MockSkillClaimMiddleware.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Security.Claims;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder.Skills;
+using Microsoft.Bot.Connector.Authentication;
+
+namespace Microsoft.Bot.Builder.Dialogs.Adaptive.Testing.Mocks
+{
+    /// <summary>
+    /// Middleware for mocking skill claim.
+    /// </summary>
+    public class MockSkillClaimMiddleware : IMiddleware
```

Comment:
I don't think we should rely on middleware to add claims for skills, we should use some other mechanism. 
This is also not a really mock, mocks imitate something.

Reply:
Right.. I see we are working on things for skill here https://github.com/microsoft/botframework-sdk/issues/5726
I will close this now and wait until the new designs come in